src/App.js

```diff
@@ -13,12 +13,14 @@ import {Theme} from "./store/reducers/reducers";
 const App = () => {
     const isDarkMode = useSelector((state) => state.Theme.darkTheme);
 
+    localStorage.setItem('isDarkMode',JSON.stringify(isDarkMode));
+    const getIsDarkMode = localStorage.getItem('isDarkMode');
```

Comment:
when you start the app, get data from local storage and put into redux as initial store